PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -3398,3 +3406,16 @@ def _replace_paths_in_code(self, co):
                         co.co_varnames, new_filename, co.co_name,
                         co.co_firstlineno, co.co_lnotab,
                         co.co_freevars, co.co_cellvars)
+
+
+def _is_namespace_package(location):
+    try:
+        with open(location, "rb") as stream:
+            data = stream.read(4096)
```

Comment:
Why 4096? I understand it was copy-pasted from astroid, but if we know what this stands for it would be great to move this number to a constant so we get a meaningful name rather than the magic number here.

Or at least a comment if this is the "expected length".

Reply:
I'm not sure why exactly it is 4096 and I was wondering myself if we should just read a few lines in text mode, but given that Astroid is so widely used, I decided to stick to their implementation.